### PR TITLE
[HIPIFY][SWDEV-316762][fix] Soft link to /opt/rocm/bin is missed for hipify-perl

### PR DESCRIPTION
**Pull request by aakanksha555**
Revised PR to include sym-links for all hipify-perl and related scripts as well as hipify-clang

### CODE REVIEW

File: packaging/hipify-clang.postinst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+# Soft-link to bin files
+HIPBINDIR="/opt/rocm/hip/bin"
+ROCMBINDIR="/opt/rocm/bin"
```
Comment (emankov) [inline packaging/hipify-clang.postinst:7]: What's the problem in using `CPACK_PACKAGING_INSTALL_PREFIX` set in CMakeLists.txt instead?

File: CMakeLists.txt [hunk 2]
```diff
@@ -115,6 +115,10 @@ if(UNIX)
     set(CPACK_PACKAGING_INSTALL_PREFIX "/opt/rocm/hip" CACHE PATH "HIP Package Installation Path")
```
Comment (emankov) [inline CMakeLists.txt:115]: I'd add the following two lines here:
```cmake
set(HIPBINDIR ${CPACK_PACKAGING_INSTALL_PREFIX}/bin)
set(ROCMBINDIR ${CPACK_PACKAGING_INSTALL_PREFIX}/../bin)
```

And remove the following two lines in the below `packaging/hipify-clang.*` files:
```cmake
HIPBINDIR="/opt/rocm/hip/bin"
ROCMBINDIR="/opt/rocm/bin"
```

Could you test it on all the target OSs, please?
Thank you!

Comment (aakanksha555) [reply CMakeLists.txt:115]: Tested it out. The post install files don't have visibility of the local variables in CMakeLists.txt.

Comment (emankov) [reply CMakeLists.txt:115]: Aha. I thought that not only `hipify-clang.txt` does have visibility of local variables declared in `CMakeLists.txt`.

**Pull request merged**